VaticanBAV/et/BAVet151.xml

```diff
@@ -89,16 +91,37 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
 <list>
 
+<item xml:id="a1">
+  <desc type="MixedNote">Names of days written in the margins:
+    <q xml:lang="gez">ዘሰኑይ፡</q> on <locus target="#1">f. 1</locus>
+    <q xml:lang="gez">ዘሠሉስ፡</q> on <locus target="#17">f. 17</locus>
+    <q xml:lang="gez">ዘረቡዕ፡</q> on <locus target="#25">f. 25</locus>
+    <q xml:lang="gez">ዘሐሙስ፡</q>on <locus target="#47">f. 47</locus>.
+  </desc>
+</item>
+
 <item xml:id="e1">
   <desc>The previous shelfmark <q xml:lang="la">Vat. | aeth. | 151</q> is written on a sheet of green leather glued to the cover, together with the coat
      of arms of pope <persName ref="PRS7896PiusXII"/> and of the librarian <persName ref="PRS13926Ehrle"/>.</desc>
 </item>
 
+<item xml:id="e2">
+  <desc>The name of the owner has been erased. On <locus target="#46v">f. 46v</locus> and <locus target="#58v">58v</locus>
+  that of <persName role="owner" ref="PRS13932Walda">Walda Ṣādǝq</persName> has been written.
+</desc>
+</item>
+
+<item xml:id="e3">
+<desc></desc>
```

Comment:
e3 is empty - if you plan to insert something later, I would just comment it out for the time being.